user/hardware/hardware-testing.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: doc
+title: Hardware Testing
+permalink: /doc/hardware-testing/
+---
+
+# Hardware Testing
+
+The Qubes developers test Qubes OS on certain hardware models.
+
+## Tested Models
+
+These hardware models have been (and continue to be) tested and work well with Qubes OS:
```

Comment:
I'd explicitly mention this is in addition to certified hardware.

Reply:
I have significantly expanded the introduction to explain the relationship between tested hardware, certified hardware, and the HCL. Please let me know what you think.

Reply:
First two points in this list reminds me hardware support levels here: https://www.qubes-os.org/news/2017/07/08/toward-a-reasonably-secure-laptop/

Reply:
Ah, indeed, but Level 0 still seems distinct from what we have here, because it comes with a guarantee of compatibility. I think it would still be good to allow vendors to submit hardware models for Level 0 certification, but I don't think we want to commit to our own testing to being Level 0.